aqt/editor.py

```diff
@@ -337,11 +345,12 @@ def oncallback(arg):
                 self.web.setFocus()
             runHook("loadNote", self)
 
-        self.web.evalWithCallback("setFields(%s); setFonts(%s); focusField(%s); setNoteId(%s)" % (
-            json.dumps(data),
-            json.dumps(self.fonts()), json.dumps(focusTo),
-                                  json.dumps(self.note.id)),
-                                  oncallback)
+        self.web.evalWithCallback(
```

Comment:
Please avoid reformatting parts of the code you're not changing - it makes it hard to review. You can use git add -p or a similar tool to selectively include content in your patch, or prevent your editor from reformatting automatically.

Reply:
This line in particular was formatted confusingly - I am reformatting it on purpose.
Previously, line 343 (`json.dumps(self.note.id)`) was aligned differently from the 2 lines above. But all the `json.dumps` should all have the same indentation level, because they are all part of the same tuple passed to the `%` operator.

But if you'd prefer to not change it, let me know and I'll revert the hunk.

Reply:
I think these tweaks are best done in bulk in an automated fashion. I had planned to hold off until the new dev branch to introduce automatic formatting, but thinking about it more, that will make it harder to port patches back and forth between the branches, so perhaps this is a bandaid that needs to be ripped off sooner rather than later. I'll merge these changes as is; please hold off on new pull requests until I get the code reformatted so you don't get conflicts.

Reply:
OK